frontend/src/__tests__/cypress/cypress/tests/mocked/application.cy.ts

```diff
@@ -113,4 +114,22 @@ describe('Application', () => {
     aboutDialog.findText().should('contain.text', 'OpenShift');
     aboutDialog.findProductName().should('contain.text', 'OpenShift AI');
   });
+  it('should show the login modal when receiving a 403 status code', () => {
```

Comment:
This test can also intercept the call to `/oauth/sign_out` when the user clicks the `Log in` button.